tests/test_decaydata.py

```diff
@@ -199,22 +199,21 @@ def test_decaydata_instantiation(self):
         self.assertEqual(data.dataset, "icrp107")
         self.assertEqual(data.hldata[0][0], 100.5)
         self.assertEqual(data.hldata[0][1], "d")
-        self.assertEqual(data.hldata[-1][0], 12.32)
-        self.assertEqual(data.hldata[-1][1], "y")
-        self.assertEqual(data.num_radionuclides, 1252)
+        self.assertEqual(data.hldata[-1][0], np.inf)
+        self.assertEqual(data.hldata[-1][1], "s")
+        self.assertEqual(data.num_radionuclides, 1498)
```

Comment:
This is something that has been playing on my mind. Now the datasets include stable nuclides, the variable naming no longer feels like a great match - right up to the "Radionuclide" class. Anyway, rather than making wholesale changes across the code now, I think keeping things as they are is ok. This package is about radioactive decay after all..